### PR TITLE
fix: Debug logging for migration handlers [DEV-1974]

### PR DESCRIPTION
**Pull request by abdulla-ashurov**
- Add migration logs for DID and resources

### CODE REVIEW

**Comment by ankurdotb** [comment]
Task linked: [DEV-1974 Log migrations for DIDs and resources if in debug mode](https://app.clickup.com/t/6600954/DEV-1974)

**Review by ankurdotb — CHANGES_REQUESTED** [review]
Log all changes that happen, not just Indy-style to new-style version migrations

**Comment by ankurdotb** [comment]
@abdulla-ashurov I've retargetted this to main

**Comment by abdulla-ashurov** [comment]
> @abdulla-ashurov I've retargetted this to main

@ankurdotb Thanks 🙏

**Comment by abdulla-ashurov** [comment]
> Log all changes that happen, not just Indy-style to new-style version migrations

@ankurdotb Done! ✅

**Review by ankurdotb — CHANGES_REQUESTED** [review]
1. **Protobufs**
    1. migrations.MigrateDidProtobuf: Old and new metadata
    2. migrations.MigrateResourceProtobuf: Nothing
2. **Indy style**
    1. `migrations.MigrateDidIndyStyle`: Old DID and new DID (not DIDDoc, just ID)  
    2. `migrations.MigrateResourceIndyStyle: Old and new Collection ID
4. **UUID normalization**
    1. migrations.MigrateDidUUID: Old DID and new DID (not DIDDoc, just ID)  
    2. migrations.MigrateResourceUUID: Old and new Collection ID
5. `migrations.MigrateDidVersionId`: DID ID, old version, new version
6. `migrations.MigrateResourceChecksum`: Collection ID, Resource ID, old and new checksum
7. `migrations.MigrateResourceVersionLinks`: Collection ID, Resource, old and new version links
8. `migrations.MigrateResourceDefaultAlternativeUrl`: Collection ID, Resource ID, Alternative URL

File: app/app.go [hunk 1]
```diff
@@ -905,6 +905,23 @@ func New(
...
+	// 	storeUpgrades := storetypes.StoreUpgrades{
+	// 		Added: []string{
+	// 			icahosttypes.StoreKey,
+	// 			icacontrollertypes.StoreKey, // <-- this is the new store key
+	// 		},
+	// 	}
+
+	// 	app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
+	// }
+
```
[the remark this answers is not in the document]
Comment (abdulla-ashurov) [reply app/app.go]: Done! ✅

**Review by ankurdotb — APPROVED** [review]

**Pull request merged**